io-uring-test/src/tests/register_sync_cancel.rs

```diff
@@ -0,0 +1,105 @@
+use std::io;
+
+use io_uring::cqueue;
+use io_uring::opcode;
+use io_uring::squeue;
+use io_uring::types::{self, AsyncCancelFlags};
+use io_uring::IoUring;
+
+use crate::Test;
+
+pub fn test_register_sync_cancel<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
+    ring: &mut IoUring<S, C>,
+    test: &Test,
+) -> io::Result<()> {
+    require!(
+        test; // We need at least 5.19 for these tests, UringCmd16 is a proxy for that.
```

Comment:
Hmm this is wrong I think we need to probe for 6.0

Reply:
I switched this to probe for `opcode::SendZc::CODE` since that seems to be available only on 6.0.